module/VuFind/src/VuFind/Db/Entity/LoginToken.php

```diff
@@ -0,0 +1,360 @@
+ * @ORM\Entity
+ */
+class LoginToken implements LoginTokenEntityInterface
+{
+    /**
+     * Unique ID.
+     *
+     * @var string
+     *
+     * @ORM\Column(name="id",
+     *          type="integer",
+     *          nullable=false
+     * )
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="NONE")
+     */
+    protected $id;
+
+    /**
+     * User ID.
+     *
+     * @var User
+     *
+     * @ORM\ManyToOne(targetEntity="VuFind\Db\Entity\User")
+     * @ORM\JoinColumns({
+     * @ORM\JoinColumn(name="user_id",
+     *              referencedColumnName="id")
+     * })
+     */
+    protected $user;
+
+    /**
+     * Token.
+     *
+     * @var string
+     *
+     * @ORM\Column(name="token",
+     *             type="string",
+     *             length=255,
+     *             nullable=false
+     * )
+     */
+    protected $token;
+
+    /**
+     * Series.
+     *
+     * @var string
+     *
+     * @ORM\Column(name="series",
+     *             type="string",
+     *             length=255,
+     *             nullable=false
+     * )
+     */
+    protected $series;
+
+    /**
+     * Last login date.
+     *
+     * @var \DateTime
+     *
+     * @ORM\Column(name="last_login",
+     *          type="datetime",
+     *          nullable=false
+     * )
+     */
+    protected $lastLogin = '2000-01-01 00:00:00';
+
+    /**
+     * Browser.
+     *
+     * @var ?string
+     *
+     * @ORM\Column(name="browser",
+     *             type="string",
+     *             length=255,
+     *             nullable=true
+     * )
+     */
+    protected $browser;
+
+    /**
+     * Platform.
+     *
+     * @var ?string
+     *
+     * @ORM\Column(name="platform",
+     *             type="string",
+     *             length=255,
+     *             nullable=true
+     */
+    protected $platform;
+
+    /**
+     * Expires.
+     *
+     * @var int
+     *
+     * @ORM\Column(name="expires",
+     *             type="integer",
+     *             nullable=false
+     * )
+     */
+    protected $expires;
+
+    /**
+     * Last session ID.
+     *
+     * @var ?string
+     *
+     * @ORM\Column(name="last_session_id",
+     *             type="string",
+     *             length=255,
+     *             nullable=true
+     */
+    protected $lastSessionId;
+
+    /**
+     * Getter for ID.
+     *
+     * @return int
+     */
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * Setter for User.
+     *
+     * @param UserEntityInterface $user User to set
+     *
+     * @return LoginTokenEntityInterface
+     */
+    public function setUser(UserEntityInterface $user): LoginTokenEntityInterface
+    {
+        $this->user_id = $user->getId();
+        return $this;
+    }
+
+    /**
+     * User getter (only null if entity has not been populated yet).
+     *
+     * @return ?UserEntityInterface
+     */
+    public function getUser(): ?UserEntityInterface
+    {
+        return $this->user_id
```

Comment:
Similarly, in this class, the object will be automatically populated, so you can simply `return $this->user;`.